canonical-web-and-design/canonical.com
# remove duplicated copy

## Done

Removed duplicated copy on /careers/lifestyle

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers/lifestyle
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [copy doc](https://docs.google.com/document/d/15Ofb6vRUXbQ3evcWmrOu9ymuY37ayg4M_xZ2QXQyIts), and that the line "Second, we’re a fully distributed organisation." isn't repeated.


## Issue / Card

Fixes #127
